centreon/packages/js-config/cypress/e2e/configuration.ts

```diff
@@ -51,7 +51,7 @@ export default ({
     },
     env: {
       ...env,
-      OPENID_IMAGE_VERSION: '23.04',
+      OPENID_IMAGE_VERSION: '23.10',
```

Comment:
develop now means 24.04

Reply:
I will try to replace this by a dynamic value to avoid to update this at each new major version